src/main/scripts/org/reaktivity/specification/amqp/link/transfer.as.receiver/server.rpt

```diff
@@ -0,0 +1,129 @@
+      [0x03]                                                      # number of fields in list = 3
+      [0xa1 0x05 0x6c 0x69 0x6e 0x6b 0x31]                        # name = "link1"
+      [0x43]                                                      # handle = 0
+      [0x42]                                                      # role = sender
+
+# flow frame
+read [0x00 0x00 0x00 0x1c]                                        # length = 28
+     [0x02]                                                       # dof = 2
+     [0x00]                                                       # type = AMQP (0)
+     [0x00 0x01]                                                  # channel = 1
+     [0x00 0x53 0x13]                                             # performative = flow (19)
+     [0xc0 0x0f]                                                  # list type (list8), physical length = 15
+     [0x04]                                                       # number of fields in list = 4
+     [0x52 0x01]                                                  # next-incoming-id = 1
+     [0x70 0x00 0x00 0x07 0xff]                                   # incoming-window = 2047
+     [0x52 0x01]                                                  # next-outgoing-id = 1
+     [0x70 0x7f 0xff 0xff 0xff]                                   # incoming-window = 2147483647
+
+# transfer frame
+write [0x00 0x00 0x00 0x0f]                                       # length = 15
+      [0x02]                                                      # dof = 2
+      [0x00]                                                      # type = AMQP (0)
+      [0x00 0x01]                                                 # channel = 1
+      [0x00 0x53 0x14]                                            # performative = transfer (20)
+      [0xc0 0x02]                                                 # list type (list8), physical length = 2
+      [0x01]                                                      # number of fields in list = 1
+      [0x43]                                                      # handle = 0
+
+# disposition frame
+read [0x00 0x00 0x00 0x10]                                        # length = 16
+     [0x02]                                                       # dof = 2
+     [0x00]                                                       # type = AMQP (0)
+     [0x00 0x01]                                                  # channel = 1
+     [0x00 0x53 0x15]                                             # performative = disposition (21)
+     [0xc0 0x03]                                                  # list type (list8), physical length = 3
+     [0x02]                                                       # number of fields in list = 2
+     [0x41]                                                       # role = receiver
+     [0x43]                                                       # first = 0
```

Comment:
Initially we want `at-most-once` message delivery, so already settled at transfer and no disposition.